src/modules/commander/Commander.cpp

```diff
@@ -327,44 +255,37 @@ int commander_main(int argc, char *argv[])
 	}
 
 	if (!strcmp(argv[1], "status")) {
-		print_status();
+		PX4_INFO("arming: %s", arming_state_names[status.arming_state]);
 		return 0;
 	}
 
 	if (!strcmp(argv[1], "calibrate")) {
 		if (argc > 2) {
-			int calib_ret = OK;
+			if (!strcmp(argv[2], "gyro")) {
+				send_vehicle_command(vehicle_command_s::VEHICLE_CMD_PREFLIGHT_CALIBRATION, 1);
```

Comment:
```suggestion
				send_vehicle_command(vehicle_command_s::VEHICLE_CMD_PREFLIGHT_CALIBRATION, 1.f, 0.f, 0.f, 0.f, 0.f, 0.f, 0.f);
```
I would fill all of these calibration call parameters up with zero float literals because you defined the default parameters to be all NAN and the calibration command handle blindly casts them to ints. According to the IEEE float standard and C/C++ standard casting a float NAN to int is undefined and you fall back to some machine-specific result, see https://stackoverflow.com/questions/10366485/problems-casting-nan-floats-to-int

Reply:
Kind of ugly, but it's probably better to keep the NANs by default for other usage.

Reply:
Done.